dashboards_bundlers/server_upload/__init__.py

```diff
@@ -87,8 +87,10 @@ def send_file(file_path, dashboard_name, handler):
             token = os.getenv('DASHBOARD_SERVER_AUTH_TOKEN')
             if token:
                 headers['Authorization'] = 'token {}'.format(token)
+            skip_verify = os.getenv('DASHBOARD_SERVER_NO_SSL_VERIFY', '').lower() in ['yes', 'true']
```

Comment:
Now logging a warning when the module is imported. Added a function to read the env var every time to aid testing, however.